packages/cluster-ui/src/statementDetails/statementDetails.tsx

```diff
@@ -431,16 +431,16 @@ export class StatementDetails extends React.Component<
                 <Row>
                   <Col>
                     <div className={summaryCardStylesCx("summary--card__item")}>
-                      <Heading type="h5">Statement time</Heading>
-                      <Text>
+                      <Heading type="h5">Mean statement time</Heading>
+                      <Text type="body-strong">
```

Comment:
nit: don't worry about it now, but in general I would've put these style changes in a separate commit